includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -369,18 +372,53 @@ protected function get_row_data( $item ) {
 					$course_title = '<strong><a class="row-title" href="' . esc_url( $url ) . '">' . apply_filters( 'the_title', $item->post_title, $item->ID ) . '</a></strong>';
 				}
 
-				$column_data = apply_filters(
-					'sensei_analysis_overview_column_data',
-					array(
-						'title'              => $course_title,
-						'last_activity'      => $last_activity_date,
-						'completions'        => $course_completions,
-						'average_percent'    => $average_grade,
-						'days_to_completion' => $average_completion_days,
-					),
-					$item,
-					$this
+				// Fetch learners in course.
+				$course_args           = array(
+					'post_id' => $item->ID,
+					'type'    => 'sensei_course_status',
+					'status'  => array( 'in-progress', 'complete' ),
+				);
+				$course_students_count = Sensei_Utils::sensei_check_for_activity( $course_args );
+
+				// Calculate average lesson progress per student for course.
+				$average_course_progress = 0;
```

Comment:
Looks like you can remove this line. You set the default value [here](https://github.com/Automattic/sensei/pull/4917/files#diff-6ce1dc075c4832d5bf050e621bcfecbab91d97504e8bc565f4f476b1ea2f654dR400).

Reply:
Removed it: [no more setting it to 0](https://github.com/Automattic/sensei/pull/4917/files#diff-6ce1dc075c4832d5bf050e621bcfecbab91d97504e8bc565f4f476b1ea2f654dR551)